### PR TITLE
SDK: Store stacktrace with each captured event

### PR DESCRIPTION
**Pull request by medikoo**

### CODE REVIEW

**Comment by medikoo** [comment]
@Danwakeem we've got a green light form @skierkowski

File: node/packages/sdk/lib/resolve-stack-trace-string.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const resolveStackString = (error) => {
+  if (!error) return null;
```
Comment (Danwakeem) [inline node/packages/sdk/lib/resolve-stack-trace-string.js:4]: Is it correct that we would store the stacktrace as `null` if we don't have an error object?

Comment (medikoo) [reply node/packages/sdk/lib/resolve-stack-trace-string.js:4]: No, in this case we fallback to locally resolved stack (this function is re-run with error instance created on spot

Comment (Danwakeem) [reply node/packages/sdk/lib/resolve-stack-trace-string.js:4]: Ahh okay awesome 😎

**Review by Danwakeem — APPROVED** [review]
:shipit:

**Pull request merged**